### PR TITLE
Feature/add es lint

### PR DESCRIPTION
**Pull request by DominikDR**
This branch will add following changes:
- PropTypes to components
- Migration from promises to async functions
- ESLint configured for the project
- ESLinted all project files.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -12,18 +12,22 @@
...
+    "babel-loader": "^8.0.4",
     "css-loader": "^1.0.0",
-    "express": "^4.16.3",
+    "eslint": "^5.9.0",
+    "eslint-config-airbnb": "^17.1.0",
+    "eslint-plugin-import": "^2.14.0",
+    "eslint-plugin-jest": "^22.1.0",
+    "eslint-plugin-jsx-a11y": "^6.1.2",
+    "eslint-plugin-react": "^7.11.1",
+    "event-stream": "3.3.4",
```
Comment (Dammic) [inline package.json]: What is that event-stream for?

File: package.json [hunk 2]
```diff
@@ -33,6 +37,12 @@
     "webpack-cli": "^3.1.1"
   },
   "dependencies": {
+    "body-parser": "^1.18.3",
```
Comment (Dammic) [inline package.json]: Where did that came from?
Shouldn't be a devdependency?

File: package.json [hunk 3]
```diff
@@ -33,6 +37,12 @@
...
+    "lodash.pick": "^4.4.0",
+    "lodash.sample": "^4.2.1",
+    "express": "^4.16.3",
```
Comment (Dammic) [inline package.json]: Should be dev dep

File: package.json (hunk 2 above)
Comment (DominikDR) [reply package.json]: That comes from server.js
[eslint] 'body-parser' should be listed in the project's dependencies, not devDependencies.

File: package.json (hunk 3 above)
Comment (DominikDR) [reply package.json]: As above, eslint says it should be dep.

File: package.json (hunk 1 above)
Comment (DominikDR) [reply package.json]: It's older version of this package due to security vulnerabilities. It's here temporary I think.

Comment (Dammic) [reply package.json]: You should try to find the package that has this vulnerability and fix it. There is no point in adding dependencies that are not directly needed by you

File: package.json (hunk 2 above)
Comment (Dammic) [reply package.json]: I would still try to fix that. You have it installed - things from dependencies will be used on frontend https://stackoverflow.com/questions/44939304/eslint-should-be-listed-in-the-projects-dependencies-not-devdependencies

File: package.json (hunk 3 above)
Comment (Dammic) [reply package.json]: like above 100%

Comment (DominikDR) [reply package.json]: [https://medium.com/@stalonadsl948/dependencies-vs-devdependencies-926e096a3dee](url)

> Dependencies should contain libs and framewors your app is built on, such as Vue, React, Angular, Express, JQuery and etc. You will agree with me, if I say, that your project wont work without these packages(if you are using them, of course).

File: package.json (hunk 2 above)
Comment (DominikDR) [reply package.json]: As below.

File: package.json (hunk 1 above)
Comment (DominikDR) [reply package.json]: But how to fix it if the latest version of this package which has a dangerous code installs automatically through "npm install"?

Comment (Dammic) [reply package.json]: Which package is that?

File: package.json (hunk 3 above)
Comment (Dammic) [reply package.json]: Just leave it as it is

File: package.json (hunk 1 above)
Comment (DominikDR) [reply package.json]: Ok I read that version is lock to 3.3.4 so I deleted it from dev dep.

**Pull request merged**